src/bicep-registry/README.md

```diff
@@ -0,0 +1,229 @@
+| `-Location`      | Optional. Azure location to execute the deployment from. Default = `westus`.                                                       |
+| `-Parameters`    | Optional. Parameters to pass thru to the deployment. Defaults per template/module are configured in the script.                    |
+| `-Build`         | Optional. Indicates whether the the `Build-Toolkit` command should be executed first. Default = `false`.                           |
+| `-Test`          | Optional. Indicates whether to run the template or module test instead of the template or module itself. Default = `false`.        |
+| `-Debug`         | Optional. Writes script execution troubleshooting details to console. Does not execute deployment.                                 |
+| `-WhatIf`        | Optional. Validates the deployment without executing it or changing resources.                                                     |
+
+<br>
+
+## Publishing bicep registry modules
+
+All modules in this folder are published to the official [Bicep Registry](https://github.com/Azure/bicep-registry-modules). If your module does not require multi-scope support, you can publish it directly to the Bicep Registry. If your module does require multi-scope support, follow the steps above for creating, building, and testing your module. Then, follow the [Bicep Registry contribution guide](https://github.com/Azure/bicep-registry-modules/blob/main/CONTRIBUTING.md) using the locally-generated and validated modules.
+
+The Bicep Registry contribution guide will have you run `brm generate` before submitting your PR. Do not commit the changes from this command back to the FinOps toolkit repo. All other manual changes to the module itself should be made within the FinOps toolkit repo.
+
+If you find a new requirement arises for Bicep Registry onboarding, please update this document and associated scripts. Please [file an issue](https://github.com/microsoft/cloud-hubs/issues) if you identify any bugs or have ideas to improve the process.
+
+<br>
+
+## Templating language
+
+Bicep offers a clean solution to define reusable modules, but has a few limitations that make it onerous for modules that support multiple scopes. We developed a very simple templating language to work around the following limitations:
+
+1. Modules cannot target multiple scopes.
+2. Scope functions behave differently for each targeted scope.
+3. No mechanism for conditional logic based on scope.
+
+Our templating language supports "scope directives" that enable you to:
+
+1. Leverage existing Bicep developer tooling for the primary scope.
+2. Target multiple scopes.
+3. Conditionally include a single line.
+4. Conditionally include a multiple lines.
+
+### Scope directives
+
+Scope directives are single-line comments that reference one or more Bicep-supported scopes. Scopes are prefixed with `@` and are separated by spaces. Do not include additional characters after the scope directive.
```

Comment:
Should we list or link to Bicep-supported scopes, I mean ln 151 is likely the list but not explicitly called out.